CHANGELOG.rst

```diff
@@ -2,9 +2,14 @@
 Release Notes
 =============
 
-Unreleased
+1.18.0
 ================
 
+- AP-2106 Add new GPG keys for APT and RPM repositories signature rotation (#44)
+- Restore usage of underscores in options as it would require change at several places and would probably bring confusion (#52)
```

Comment:
This is a fix to another change in the same version (#51), and therefore doesn't need to be shown in the changelog.